<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HSpecFittingLinearWidget</class>
 <widget class="QWidget" name="HSpecFittingLinearWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>132</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_001">
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>取样次数：</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox_1"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_004">
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>基准像元位置：</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_2"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_002">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>起始积分时间：</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_1"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_003">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>终止积分时间：</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
